parser_library/src/processing/processing_manager.cpp

```diff
@@ -250,17 +250,41 @@ void processing_manager::finish_copy_member(copy_processing_result result)
     lsp_analyzer_.copydef_finished(member, std::move(result));
 }
 
+namespace {
+void add_preproc_ids(const semantics::preproc_details& details, context::id_storage& ids)
+{
+    if (!details.label.name.empty())
```

Comment:
I don't understand this function. It adds various strings into the id_storage, but ignores the returned id_index values.

Reply:
This is related to the comment you've had just below this one - I didn't want the lsp_analyzer to `add()` stuff but it's probably an non-issue. Changed.